examples/tools/autotools/autotools_toolchain/build_project_autotools_toolchain.rst

```diff
@@ -0,0 +1,214 @@
+.. _examples_tools_autotools_autotools_toolchain_build_project_autotools_toolchain:
+
+Build a simple Autotools project using Conan
+============================================
+
+In this example, we are going to create a string formatter application
+that uses one of the most popular C++ libraries: `fmt <https://fmt.dev/latest/index.html/>`__.
+
+We'll use `Autotools <https://www.gnu.org/software/automake/manual/html_node/Autotools-Introduction.html>`_ as build system and `pkg-config <https://www.freedesktop.org/wiki/Software/pkg-config/>`_ as helper tool in this case, so you should get them installed
+on Linux and Mac before going forward with this example.
+
+Please, at first, clone the sources to recreate this project, you can find them in the
+`examples2.0 repository <https://github.com/conan-io/examples2>`_ in GitHub:
+
+.. code-block:: shell
+
+    git clone https://github.com/conan-io/examples2.git
+    cd examples2/examples/tools/autotools/autotoolstoolchain/string_formatter
+
+We start from a very simple C++ language project with this structure:
+
+.. code-block:: text
+
+    .
+    ├── configure.ac
+    ├── Makefile.am
+    ├── conanfile.txt
+    └── src
+        └── main.cpp
+
+This project contains a basic `configure.ac <https://www.gnu.org/software/autoconf/manual/autoconf-2.60/html_node/Writing-configure_002eac.html>_` including the **fmt** pkg-config dependency and the
+source code for the string formatter program in *main.cpp*.
+
+Let's have a look at the *main.cpp* file, it only prints a default message, but uses ``fmt::print`` method for it.
```

Comment:
```suggestion
Let's have a look at the *main.cpp* file, it only prints a simple message, but uses ``fmt::print`` method for it.
```
I don't thing `default` is right word here

Reply:
fixed, thanks!